jcache/commons/src/main/java/org/infinispan/jcache/Closeables.java

```diff
@@ -0,0 +1,40 @@
+package org.infinispan.jcache;
+
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.jcache.logging.Log;
+
+import java.io.Closeable;
+import java.util.Set;
+
+class Closeables {
+
+   private static final Log log = LogFactory.getLog(Closeables.class, Log.class);
+
+   Closeables() {
+   }
+
+   static void close(Object o) {
```

Comment:
Out of curiosity - why this method doesn't take `Closeable` as parameter but `Object` and checks its class? If you want to make this very generic, you could take an object and check if it contains `close` method rather than doing type check.

Reply:
If you look at how it's used, it's invoked on `ExpiryPolicy`, `CacheLoader` and `CacheWriter` which none of those extend `Closeable`, but the TCK expects that if implementations extend `Closeable`, they need to be closed. I think check for the `close` method would be a bit over the top of what's required here.